src/harvest.cpp

```diff
@@ -119,13 +122,30 @@ void harvest_list::check_consistency()
 {
     for( const auto &pr : harvest_all ) {
         const auto &hl = pr.second;
+        const std::string hl_id = hl.id().c_str();
+        auto error_func = [&]( const harvest_entry & entry ) {
+            std::string errorlist = "";
+            if( !( item::type_is_defined( entry.drop ) || ( entry.type == "bionic_group" &&
+                    item_group::group_is_defined( entry.drop ) ) ) ) {
+                errorlist += entry.drop;
+            }
+            // non butchery harvests need to be excluded
+            if( std::string( hl_id.substr( 0, 14 ) ) != std::string( "harvest_inline" ) ) {
```

Comment:
```suggestion
            if( hl_id.substr( 0, 14 ) != "harvest_inline" ) {
```
`std::string::substr` already returns a `string` and a `string` can be compared with a string literal directly.